### PR TITLE
Automated Stubbed test case for Settings feature.

### PR DESCRIPTION
**Pull request by shubhamsg199**
Automated stubbed test case **test_positive_update_email_reply_address** in test_setting.py

```
pytest -k test_positive_update_email_reply_address tests/foreman/cli/test_setting.py 
============================ test session starts ============================
platform linux -- Python 3.8.8, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/sganar/satellite/robottelo, configfile: pyproject.toml
plugins: services-2.2.1, mock-3.6.1, reportportal-5.0.8, forked-1.3.0, xdist-2.3.0, ibutsu-1.16, cov-2.12.1
collected 30 items / 29 deselected / 1 selected                                                                                                                                                                                           

tests/foreman/cli/test_setting.py .                                                                                                                                                                                                 [100%]

=============== 1 passed, 29 deselected, 3 warnings in 31.27s================
```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

File: tests/foreman/cli/test_setting.py [hunk 1]
```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
 
     :CaseImportance: Low
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
     """
+    sendmail_argument_value = gen_string('alphanumeric')
+    arguments = {
```
Comment (devendra104) [inline tests/foreman/cli/test_setting.py:232]: `email_config_params` or `sendmail_config_params` would be much appropriate than `arguments`.

File: tests/foreman/cli/test_setting.py [hunk 2]
```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
...
+        'delivery_method': "sendmail",
```
Comment (devendra104) [inline tests/foreman/cli/test_setting.py:233]: use single quotes.
```suggestion
        'delivery_method': 'sendmail',
```

File: tests/foreman/cli/test_setting.py [hunk 3]
```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
...
+        'sendmail_arguments': f'{sendmail_argument_value}',
+        'sendmail_location': "/usr/sbin/sendmail",
```
Comment (devendra104) [inline tests/foreman/cli/test_setting.py:235]:
```suggestion
        'sendmail_location': '/usr/sbin/sendmail',
```

**Comment by devendra104** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_update_email_delivery_method_sendmail' tests/foreman/cli/test_setting.py

File: tests/foreman/cli/test_setting.py [hunk 4]
```diff
@@ -226,8 +226,17 @@ def test_positive_update_email_delivery_method_sendmail():
 
     :CaseImportance: Low
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
     """
```
Comment (shweta83) [inline tests/foreman/cli/test_setting.py:230]: Please add "CaseLevel, Steps and expectedresults"

Comment (akhil-jha) [reply tests/foreman/cli/test_setting.py:230]: Steps and expectedresults are already there. They were hiding :grin:

Comment (shubhamsg199) [reply tests/foreman/cli/test_setting.py:230]: yes, I'll add the CaseLevel

Comment (devendra104) [reply tests/foreman/cli/test_setting.py:230]: CaseLevel is already mentioned at the top of the file, so I think we no need to mention it as test case level.

Comment (shubhamsg199) [reply tests/foreman/cli/test_setting.py:230]: Okay.

File: tests/foreman/cli/test_setting.py (hunk 1 above)
Comment (shubhamsg199) [reply tests/foreman/cli/test_setting.py]: Done

**Comment by akhil-jha** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_update_email_delivery_method_sendmail' tests/foreman/cli/test_setting.py

File: tests/foreman/cli/test_setting.py (hunk 2 above)
Comment (shubhamsg199) [reply tests/foreman/cli/test_setting.py]: Done

**Review by akhil-jha — APPROVED** [review]

**Review by devendra104 — APPROVED** [review]

**Pull request merged**